packages/atlas/src/hooks/useDeepMemo.ts

```diff
@@ -0,0 +1,21 @@
+import { equal as isEqual } from '@wry/equality'
```

Comment:
I think `lodash` exports `isEqual` as well, can we use that, since it's already our direct dependency?